### DISCUSSION TITLE
learning-to-code-in-python

### DISCUSSION THREAD

**Discussion by giscus** [scidroid/portfolio#4] [General]
# learning-to-code-in-python

Learn to program in Python, a high-level, interpreted, multi-paradigm language used for Web with Flask and Django, Artificial Intelligence and Data Science.

https://scidroid.co/learning-to-code-in-python

**Comment by sebaspv** [comment]
Thanks! This article is really good for a beginner like me. Any resources for further learning you’d recommend?

**Reply by scidroid** [reply]
This is a really good material to start: 
https://dev.to/scidroid/learning-to-code-in-python-the-basics-ib3

**Reply by scidroid** [reply]
cool
